lib/rules/web/admin_console/4.4/common_ui_elements.xyaml

```diff
@@ -92,6 +92,15 @@ click_radio_input:
     selector:
       xpath: //input[@type='radio' and @value='<radio_text>']
     op: click
+click_close_button:
```

Comment:
It's better to name it as `click_close_icon` in case we need to add an action called `click_close_button` which has definition of `//button[text()='Close']`